bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeErrorEventValidationTest.java

```diff
@@ -49,6 +49,18 @@ public static Object[][] parameters() {
             .done(),
         singletonList(expect(ErrorEventDefinition.class, "Must reference an error"))
       },
+      {
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .serviceTask("task", t -> t.zeebeJobType("type"))
+            .boundaryEvent("catch", b -> b.error("= error"))
+            .endEvent()
+            .done(),
+        singletonList(
+            expect(
+                ErrorEventDefinition.class,
+                "The errorCode of the error catch event is not allowed to be an expression"))
+      },
```

Comment:
🔧 Let's also add the scenario of the error start event, like you had in the previous test class.